features/in_onderzoek.feature

```diff
@@ -12,109 +12,235 @@ Functionaliteit: in onderzoek
+      | volledigeNaam             | Arnitta |
+      En heeft de persoon een 'ouder' met de volgende gegevens
+      | naam                              | waarde |
+      | naam.voornamen                    | Noa    |
+  	  | naam.voorletters                  | N.     |
+      | geslacht.code                     | V      |
+      | geslacht.omschrijving             | vrouw  |
+  	  | ouderAanduiding                   | 1      |
+      En heeft de persoon een 'ouder' met de volgende gegevens
+      | naam                              | waarde |
+      | naam.voornamen                    | Jan    |
+      | naam.voorletters                  | J.     |  
+      | geslacht.code                     | M      |  
+      | geslacht.omschrijving             | man    |
+      | ouderAanduiding                   | 2      |
+      En heeft de persoon een 'partner' met de volgende gegevens
+      | naam                              | waarde          |
+      | burgerservicenummer               | 000000103       |
+      | naam.voornamen                    | Karel           |
+      | geslacht.code                     | M               |
+      | geslacht.omschrijving             | man             |
+      | aangaanHuwelijkPartnerschap.type  | Datum           |
+      | aangaanHuwelijkPartnerschap.datum | 2009-11-02      |
+      | aangaanHuwelijkPartnerschap.datum | 2 november 2009 |
+
+    Scenario: vragen om voorvoegsel zonder waarde bij groep naam in onderzoek
+      Gegeven een persoon heeft de volgende 'persoon' gegevens
+      | burgerservicenummer (01.20) | voornamen (02.10) | geboortedatum (03.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | 
+      | 000000115                   | Arnitta           | 19231213              | 010200                          | 20120920                       |
+      En de persoon heeft een ouder '1' met de volgende gegevens
+      | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | Noa               | V                           |
+      En de persoon heeft een ouder '2' met de volgende gegevens
+      | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | Jan               | M                           |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | burgerservicenummer (01.20) | voornamen (02.10 | geslachtsaanduiding (04.10) | datum aangaan (06.10) |
+      | 000000127                   | Karel            | M                           | 20091102              |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                           |
+      | type                | RaadpleegMetBurgerservicenummer  |
+      | burgerservicenummer | 000000115                        |
+      | fields              | ouders,partners,naam.voorvoegsel |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam                           | waarde  |
+      | inOnderzoek.voornamen     | true    |
```

Comment:
niet correct. er is alleen gevraagd om voorvoegsel, dus ik verwacht alleen inOnderzoek.voorvoegsel en inOnderzoek.datumIngangOnderzoek

Reply:
Ok, deze opmerking maakt veel duidelijk. Zou de Rule in regel 15 dan niet beter als volgt gedefinieerd kunnen worden:

` Rule: wanneer een gegeven uit een categorie gevraagd is en er zit onderzoek op die categorie, dan wordt ook inOnderzoek voor dat veld geleverd`

en kan regel 17 dan niet komen te vervallen?

Reply:
> Ok, deze opmerking maakt veel duidelijk. Zou de Rule in regel 15 dan niet beter als volgt gedefinieerd kunnen worden:
> 
> ` Rule: wanneer een gegeven uit een categorie gevraagd is en er zit onderzoek op die categorie, dan wordt ook inOnderzoek voor dat veld geleverd`

Dat lijkt mij idd beter.
> 
> en kan regel 17 dan niet komen te vervallen?
dat kan ik niet beoordelen.

Reply:
Is aangepast in #1269 .